src/main/java/com/online/book/store/dto/request/BookRequestDto.java

```diff
@@ -30,4 +31,6 @@ public class BookRequestDto {
 
     private String coverImage;
 
+    private Set<CategoryRequestDto> categories;
```

Comment:
```suggestion
    private Set<String> categoryIds;
```
we should add to book existed categories